src/FrequencyMapAgent.cpp

```diff
@@ -364,22 +372,26 @@ namespace geopm
     std::vector<std::pair<std::string, std::string> > FrequencyMapAgent::report_host(void) const
     {
         std::vector<std::pair<std::string, std::string> > result;
-        std::ostringstream oss;
-        oss << std::setprecision(M_PRECISION) << std::scientific;
-        for (const auto &region : m_hash_freq_map) {
-            oss << "\n    0x" << std::hex << std::setfill('0') << std::setw(16) << std::fixed;
-            oss << region.first;
-            oss << std::setfill('\0') << std::setw(0) << std::scientific;
-            oss << ": " << region.second;
-        }
+
+        std::map<uint64_t, double> full_map(m_hash_freq_map);
         for (const auto &region : m_default_freq_hash) {
-            oss << "\n    0x" << std::hex << std::setfill('0') << std::setw(16) << std::fixed;
-            oss << region;
-            oss << std::setfill('\0') << std::setw(0) << std::scientific;
-            oss << ": " << m_default_freq;
+            full_map.insert({region, m_default_freq});
+        }
+
+        std::map<std::string, Json> temp_map;
+        for (const auto &region : full_map)
+        {
+            std::ostringstream key_ss;
+            key_ss << "0x" << std::hex << std::setfill('0') << std::setw(16) << std::fixed;
+            key_ss << region.first;
+            std::string key_string = key_ss.str();
+            key_string.erase(std::remove(key_string.begin(), key_string.end(), '"'), key_string.end());
```

Comment:
Is this line still needed?  We want the keys (which are region hashes in this case) to be treated as strings.  Please add a comment here on why this line is still needed if it is.

Reply:
This line removes the extra `"` characters from the string. Previously the keys used to be strings surrounded with quotes: `"0x01234567"`